andbible/mods.d/HISB.conf

```
@@ -4,7 +4,7 @@ Lang=he
 Encoding=UTF-8
 Direction=RtoL
 Version=1.3
```

Version number should be incremented
```suggestion
Version=1.4
```